### PR TITLE
Adding PR tester jenkinsfile

### PR DESCRIPTION
**Pull request by msajidmansoori12**
Signed-off-by: M Sajid Mansoori <mmansoor@localhost.localdomain>

### CODE REVIEW

File: PR_TESTER.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                    sh "pipenv run pip install -r requirements.txt"
+                    env.TACKLE_URL = "http://"+sh(
+                        script : 'minikube ip',
+                        returnStdout: true
+                    ).trim()
+                    
+                    ocp.pollRouteUntilReady(env.TACKLE_URL,15,30)
+                    repo.checkout("https://gitlab.cee.redhat.com/mta-qe/tackle-api-tests-config.git","main")
+                    sh "sleep 40s" //Grace period after the application is serving correctly.
+                    env.keycloakAdminPassword = sh(
```
Comment (mguetta1) [inline PR_TESTER.jenkinsfile:64]: @msajidmansoori12 where do you use this variable?

File: PR_TESTER.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        }
+    }
+    post{
+        always{
+            script{
+                cleanWs()
+            }
+        }
+    }
+}
```
Comment (mguetta1) [inline PR_TESTER.jenkinsfile:80]: @msajidmansoori12 new line is missing at the end

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: PR_TESTER.jenkinsfile (hunk 1 above)
Comment (msajidmansoori12) [reply PR_TESTER.jenkinsfile:64]: @mguetta1 I saw that you are exporting it in github actions as well https://github.com/konveyor/tackle-api-tests/blob/main/.github/workflows/pull-request.yaml#L92 , that's why I added it.

File: PR_TESTER.jenkinsfile (hunk 2 above)
Comment (msajidmansoori12) [reply PR_TESTER.jenkinsfile:80]: @mguetta1 sorry I'm not sure what you mean , do i need to add new line at the end ?

File: PR_TESTER.jenkinsfile (hunk 1 above)
Comment (mguetta1) [reply PR_TESTER.jenkinsfile:64]: @msajidmansoori12 it is a leftover from tackle-ui workflow (I copied it). I should remove it

File: PR_TESTER.jenkinsfile (hunk 2 above)
Comment (mguetta1) [reply PR_TESTER.jenkinsfile:80]: @msajidmansoori12 yes, new empty line. When it is missing git complains about: "No newline at end of file"

File: PR_TESTER.jenkinsfile (hunk 1 above)
Comment (msajidmansoori12) [reply PR_TESTER.jenkinsfile:64]: @mguetta1 thanks , I'll remove it as well.

File: PR_TESTER.jenkinsfile (hunk 2 above)
Comment (msajidmansoori12) [reply PR_TESTER.jenkinsfile:80]: @mguetta1 I see , I'll add the new line.

**Review by mguetta1 — APPROVED** [review]
Looks good to me. Thanks

**Review by sshveta — APPROVED** [review]

**Pull request merged**